modal/stub.py

```diff
@@ -846,10 +848,30 @@ async def interactive_shell(self, cmd=None, image=None, **kwargs):
         async with self.run():
             await wrapped_fn.call(cmd)
 
+    def _make_remote_cls(
```

Comment:
minor preference for keeping this out of `Stub` (since it's a big class and does a lot of different things) but we can move it later since it's not a public method